module/renderer/src/commonTest/kotlin/org/cru/godtools/shared/renderer/content/BaseRenderContentTest.kt

```diff
@@ -75,4 +76,66 @@ abstract class BaseRenderContentTest {
         }
     }
     // endregion Clickable
+
+    @Test fun `IsGone`() = runComposeUiTest {
+        state.setVar("gone", listOf("value"))
+
+        setContent {
+            RenderContentStack(
+                listOf(testModel),
+                state = state,
+            )
+        }
+
+        testScope.runTest {
+            turbineScope {
+                assertTrue(testModel.isGone(state = state))
+            }
+        }
+    }
+
+    @Test fun `IsInvisible`() = runComposeUiTest {
```

Comment:
this test isn't needed because there isn't a way to check at the UI level if something is visible or not, and testing the functionality of `.isInvisible()` on the model is already handled in the parser tests